src/character_attire.cpp

```diff
@@ -1818,19 +1818,26 @@ bool outfit::can_pickVolume( const item &it, const bool ignore_pkt_settings,
     return false;
 }
 
+static void add_overlay_id_or_override( const item &item, std::vector<std::pair<std::string, std::string>> &overlay_ids ) {
```

Comment:
extracted into this helper function, do not repeat the code twice. Not sure the best approach for that